src/components/films/components/filmsList/store/sagas.ts

```diff
@@ -0,0 +1,25 @@
+import {call, put, takeEvery, delay} from 'redux-saga/effects';
+import {GET_MOVIES_START} from './types';
+import {baseUrl} from "../../../../../url";
+import {getMoviesDataFailure, getMoviesDataSuccess} from "./actions";
+
+
+function* fetchMovies() {
+  try {
+    yield delay(1000);
+    const res = yield call(fetch, baseUrl);
+    const data = yield res.json();
+    yield put(getMoviesDataSuccess(data));
+
+  } catch (e) {
+    yield put(getMoviesDataFailure(e.message));
+  }
+}
+
+
+function* watchFetchMovies() {
+  yield takeEvery(GET_MOVIES_START, fetchMovies);
```

Comment:
For multiple fetches started (*almost*) simultaneously, you may do [`takeLatest`](https://redux-saga.js.org/docs/api/#takelatestpattern-saga-args), since the result would be just the same and you would cancel all the previously fired requests.